build-tools/automation/azure-pipelines-nightly.yaml

```diff
@@ -280,3 +280,55 @@ stages:
         artifactName: Test Results - Localization With Emulator - macOS-$(System.JobPositionInPhase)
 
     - template: yaml-templates/fail-on-issue.yaml
+
+
+- stage: compliance_scan
+  displayName: Compliance
+  dependsOn: mac_build
+  jobs:
+  - job: api_scan
+    displayName: API Scan
+    pool:
+      name: Azure Pipelines
+      vmImage: windows-2022
+    timeoutInMinutes: 480
+    workspace:
+      clean: all
+    variables:
+    - name: ApiScan.Enabled
+      value: true
+    steps:
+    - template: yaml-templates/setup-test-environment.yaml
+      parameters:
+        installApkDiff: false
+        installLegacyDotNet: false
+        restoreNUnitConsole: false
+        updateMono: false
+
+    ### Copy .dll and .pdb files for APIScan
+    - task: CopyFiles@2
+      displayName: Collect Files for APIScan
+      inputs:
+        Contents: $(System.DefaultWorkingDirectory)\bin\$(XA.Build.Configuration)\dotnet\packs\Microsoft.Android*\**\?(*.dll|*.pdb)
+        TargetFolder: $(Build.StagingDirectory)\apiscan
+        OverWrite: true
+        flattenFolders: true
+      condition: and(succeeded(), eq(variables['ApiScan.Enabled'], 'true'), eq(variables['Build.SourceBranch'], 'refs/heads/dev/pjc/nightly-apiscan'))
```

Comment:
I suspect this is leftover from testing: `refs/heads/dev/pjc/nightly-apiscan`.

Reply:
Thanks, forgot to update these test conditions